db/schema.rb

```diff
@@ -24,6 +52,13 @@
     t.index ["user_id"], name: "index_tokens_on_user_id"
   end
 
+  create_table "user_calendars", force: :cascade do |t|
```

Comment:
oh it looks like this is the 'membership' table, I think you can name the table membership or user_calendar. You decide.